modules/perception/lidar/lib/detection/lidar_point_pillars/point_pillars_detection.cc

```diff
@@ -13,15 +13,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *****************************************************************************/
+#include "modules/perception/lidar/lib/detection/lidar_point_pillars/point_pillars_detection.h"
+
 #include <cuda_runtime_api.h>
+
```

Comment:
I'm afraid cpplint will complain  header files' order here. `cuda_runtime_api.h` should be placed below std headers and before other project related headers.

Reply:
> I'm afraid cpplint will complain header files' order here. `cuda_runtime_api.h` should be placed below std headers and before other project related headers.

Got it, thanks.